src/scoped.component.js

```diff
@@ -9,21 +9,35 @@ const reactSupportsReturningArrays = !!ReactDOM.createPortal;
 
 export class Scoped extends React.Component {
   static propTypes = {
-    css: PropTypes.string.isRequired,
+    css: PropTypes.string,
```

Comment:
the `css` prop is not required anymore because you can use the `postcss` prop instead

i'm checking for props in the constructor instead